packages/stylelint-config-airbnb/index.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+  plugins: [
+    'stylelint-scss',
+    'stylelint-order',
+  ],
+  rules: {
+    // Base rules
+    indentation: 2,
+    'number-leading-zero': 'never',
+    'string-quotes': 'double',
+    'selector-max-id': 0,
+    'selector-list-comma-newline-after': 'always',
+    'rule-empty-line-before': [
+      'always',
+      { ignore: ['after-comment'] },
+    ],
+    'comment-empty-line-before': [
+      'always',
+      { except: ['first-nested'] },
+    ],
+    'block-opening-brace-space-before': 'always',
+    'declaration-colon-space-after': 'always',
+    'declaration-colon-space-before': 'never',
+    'declaration-block-single-line-max-declarations': 1,
+    'declaration-property-value-blacklist': { '/^border/': ['none'] },
```

Comment:
what about it? why so?

Reply:
You can find this recommendation in the README under CSS > Border: https://github.com/airbnb/css#border

And for the 'why' part:
[This codepen](https://codepen.io/denilsonsa/pen/LkdHh) shows why `border: 0;` is supperior to `border: none;`, if your intention in applying this rule is it to _remove_ the border on the given element.